ciphers/caeser_cipher.c

```diff
@@ -0,0 +1,76 @@
+// Code by Rishabh Patel
```

Comment:
/**
 * @file
 * \brief Find approximate solution for \f$f(x) = 0\f$ using
 * Newton-Raphson interpolation algorithm.
 *
 * \author [Krishna Vedala](https://github.com/kvedala)
 */